cypress/support/commands.js

```diff
@@ -23,3 +25,96 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('fillForm', (userData) => {
+  // enter firstname:
```

Comment:
remove comments

Reply:
fixed

Reply:
I have used comments for better maintainability and readability. Or is this not a very good approach?

Reply:
That’s usually what we do, you’re right, most likely the mentor meant that in this situation it’s a little unnecessary, since you have a clear command name, and in general all the actions are quite simple and understandable

Reply:
Got it)